app/views/users/registrations/new.html.erb

```diff
@@ -1,51 +1,136 @@
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+<style type="text/css">
+  .custom-input,
+  select {
+    width: 60% !important;
+    background: transparent;
+    border: none;
+    color: #1C1C1C;
+    border-bottom: 1px solid #42B983;
+    -webkit-box-shadow: none;
+    box-shadow: none;
+    border-radius: 0;
+  }
+
+  .custom-input:focus,
+  select:focus {
+    -webkit-box-shadow: none;
+    box-shadow: none;
+    border-bottom: 1px solid #42B983;
+  }
+
+  .custom-input:-webkit-autofill,
+  .custom-input:-webkit-autofill:hover, 
+  .custom-input:-webkit-autofill:focus {
+    border-bottom: 1px solid #42B983;
+    -webkit-text-fill-color: #1C1C1C;
+    -webkit-box-shadow: 0 0 0px 1000px #fff inset;
+    transition: background-color 5000s ease-in-out 0s;
+  }  
+</style>
+
+<script type="text/javascript">
```

Comment:
It'd be a good idea to move UI relevant CSS to `/public/js/UX.js` or make a different file for it.